.github/package-template.yml

```
@@ -21,9 +22,53 @@ on:
       - vendor/%PACKAGE_NAME%/**
       - .github/workflows/%PACKAGE_NAME%.yml
 
+  schedule:
+    # Update packages nightly
+    - cron:  '0 0 * * *'
+
 jobs:
+  update-packages:
+    if: github.event_name == 'schedule'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

```suggestion
      - uses: actions/checkout@v2
```

We can use v2 eveywhere